Maestro/subscriptions.json

```diff
@@ -690,16 +695,39 @@
           "ScriptFileName": "scripts\\Invoke-UpdateDependencies.ps1",
           "Arguments": [
             "-CleanupDocker",
-              "--github-user:dotnet-maestro-bot",
-              "--github-email:dotnet-maestro-bot@microsoft.com",
-              "--github-password:`$Secrets[`'BotAccount-dotnet-maestro-bot-PAT`']",
-              "--branch:dev",
-              "--build-info-url:https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/product/cli/release/2.1/build.xml"
+            "--github-user:dotnet-maestro-bot",
+            "--github-email:dotnet-maestro-bot@microsoft.com",
+            "--github-password:`$Secrets[`'BotAccount-dotnet-maestro-bot-PAT`']",
+            "--branch:dev",
+            "--build-info-url:https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/product/cli/release/2.1/build.xml"
           ]
         },
         "vsoSourceBranch": "dev"
       }
     },
+    // Update dependencies in the aspnetcore repo's dev branch
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/master/packages.semaphore"
+      ],
+      "action": "aspnet-dependency-update",
+      "delay": "00:05:00",
+      "actionArguments": {
+        "vsoSourceBranch": "master",
+        "vsoBuildParameters": {
+          "ScriptFileName": "scripts\\UpdateDependencies.ps1",
+          "Arguments": [
+            "-BuildXml",
+            "https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/product/cli/master/build.xml",
+            "-ConfigVars '",
+            "--GithubUsername dotnet-maestro-bot",
+            "--GithubEmail dotnet-maestro-bot@microsoft.com",
+            "--GithubToken `$(`$Secrets[`'BotAccount-dotnet-maestro-bot-PAT`'])",
```

Comment:
Looking at this more closely than I have in the past, there are some problems with the secrets in these aspnet executor defs. The [aspnet-dependency-update def](https://devdiv.visualstudio.com/DevDiv/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=8517) doesn't have any secrets listed on the Variables page, and only the old-named variables are passed in by the build step. The [aspnet-docker-general def](https://devdiv.visualstudio.com/DevDiv/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=8550) does have the secrets defined in variables, but they aren't passed into the script.

These defs generally don't use azure key vault yet: https://github.com/dotnet/core-eng/issues/2382.

Reply:
You're saying that wherever we have `BotAccount-dotnet-maestro-bot-PAT` it should be `DotNetMaestroBotGitHubToken`? Or that we need to include `BotAccount-dotnet-maestro-bot-PAT` as a variable on our DevDiv definitions?

Reply:
Either way will work (and can be changed later). Sticking with `DotNetMaestroBotGitHubToken` seems like the easier change for now--IMO the name fixup can wait until we actually use key vault.

There are more steps needed than that, though: the aspnet-dependency-update def is missing the variables entirely (neither name is defined). I forget how that was handled for aspnet-docker-general--@natemcmaster, did you end up getting access to the vault? I can grab them if necessary if you let me know which naming scheme you want to go with for now.

Reply:
Yes, I got access to the keyvault so I can setup the parameters. When I setup aspnet-docker, I was told that going forward we want to keep VSTS variables and KeyVault secret names aligned where possible. @ryanbrandenburg come find me when you're free and we can setup those variables.

Reply:
Thanks. About "only the old-named variables are passed in by the build step": this applies to both defs, and to be more specific it's the params in the `PowerShell Script` step:

```
-ScriptFileName "$(ScriptFileName)" -Arguments "$(Arguments)" -Secrets @{"DotNetBotGitHubPassword"="$(DotNetBotGitHubPassword)";"DotNetMaestroBotGitHubToken"="$(DotNetMaestroBotGitHubToken)"}
```

That dictionary needs to be updated so the subscription commands can access them.